### ISSUE TITLE
Current Version 1.5.1 not able to run

### ISSUE THREAD

**Issue by janm84** [nikclayton/android-squeezer#476]
Hi,

Since the last update to current version 1.5.1 from Google Appstore, the app is not able to run on Android 9 (Huawei P20 Pro).

Please tell me if you need further information. 

Best regards 

Jan

**Comment by kampnerj** [comment]
Same behavior on pixel 2xl.

**Comment by taterpeel** [comment]
also seeing this on a samsung note 8 and a samsung tab A

**Comment by kaaholst** [comment]
I can confirm an issue where Squeezer cannot start on devices running Android 9, and where [SqueezerPlayer](https://play.google.com/store/apps/details?id=de.bluegaspode.squeezeplayer) is also installed.

I will see if I can make a fix for this.

In the meantime, here is a workaround:

1. Start Squeezeplayer manually
1. Start Squeezer
1. Select the options menu (the 3 dots in the top right corner)
1. Select settings
1. Uncheck "Start Squeezeplayer"

**Comment by michaeloreillyintel** [comment]
Yes, Android's new service restrictions cause the app to crash when a service is attempted to be started here 

    private void startControllingSqueezePlayer() {
        context.startService(getSqueezePlayerIntent());
        sendMessageDelayed(obtainMessage(MSG_TIMEOUT), TIMEOUT_DELAY);
    }


you get IllegalStateException "not allowed to start service Intent {.......PlaybackService} app is in background uid null "


https://stackoverflow.com/questions/46445265/android-8-0-java-lang-illegalstateexception-not-allowed-to-start-service-inten?rq=1
